ci/travis/determine_tests_to_run.py

```diff
@@ -54,6 +69,10 @@ def list_changed_files(commit_range):
                 RAY_CI_RLLIB_AFFECTED = 1
                 RAY_CI_LINUX_WHEELS_AFFECTED = 1
                 RAY_CI_MACOS_WHEELS_AFFECTED = 1
+            elif changed_file.startswith("python/ray/experimental/serve"):
+                RAY_CI_SERVE_AFFECTED = 1
```

Comment:
We also need to add `RAY_CI_SERVE_AFFECTED = 1` to the `python` block, the `src` bloc, and both `else` blocks.